rpm/valgrind.spec

```diff
@@ -27,6 +27,13 @@ malloc/new/free/delete are intercepted. As a result, Valgrind can
 detect a lot of problems that are otherwise very hard to
 find/diagnose.
 
+%package extratools
```

Comment:
Shouldn't this be dependent on Perl?
If so, then tools-perl might be a more descriptive name.